### PR TITLE
Improve the about page

### PR DESCRIPTION
**Pull request by Linerly**
## What does this PR do?

This PR improves the about page in SearXNG

## Why is this change important?

This fixes the grammar, spelling, and some typographical changes as well in the about page

## How to test this PR locally?

Just run SearXNG, then go to the about page in the browser (/help/en/about)

## Related issues

#1045

### CODE REVIEW

**Review by dalf — APPROVED** [review]

File: searx/infopage/en/about.md [hunk 1]
```diff
@@ -4,39 +4,39 @@ SearXNG is a fork from the well-known [searx] [metasearch engine], aggregating
...
-* SearXNG may not offer you as personalised results as Google, but it doesn't
+* SearXNG may not offer you as personalized results as Google, but it doesn't
   generate a profile about you.
 
 * SearXNG doesn't care about what you search for, never shares anything with a
-  third party, and it can't be used to compromise you.
+  third-party, and it can't be used to compromise you.
 
-* SearXNG is free software, the code is 100% open and you can help to make it
+* SearXNG is free software, the code is 100% open, and you can help to make it
```
Comment (return42) [inline searx/infopage/en/about.md]: I am not a native english speaker and my english is quit horrible, but I have a question: is it OK to add a comma before `, and`?

File: searx/infopage/en/about.md [hunk 2]
```diff
@@ -4,39 +4,39 @@ SearXNG is a fork from the well-known [searx] [metasearch engine], aggregating
...
   better.  See more on [SearXNG sources]({{GIT_URL}}).
 
 If you do care about privacy, want to be a conscious user, or otherwise believe
 in digital freedom, make SearXNG your default search engine or run it on your
-own server
+own server!
 
-## Technical details - How does it work?
+## The technical details—how does it work?
```
Comment (return42) [inline searx/infopage/en/about.md]: Lets stay with the minus and the left & right spaces.

Same as above: The Markdown parser we use ATM does not support Smart punctuation / Simple typographic replacements

File: searx/infopage/en/about.md [hunk 3]
```diff
@@ -4,39 +4,39 @@ SearXNG is a fork from the well-known [searx] [metasearch engine], aggregating
 the results of other {{link('search engines', 'preferences')}} while not
 storing information about its users.
 
-More about SearXNG ...
+More about SearXNG…
```
Comment (return42) [inline searx/infopage/en/about.md]: Lets use `:` instead of ellipsis

TLDR: The Markdown parser we use ATM does not support Smart punctuation / Simple typographic replacements 

----

@dalf I opened an issue at [mistletoe](https://github.com/miyuchina/mistletoe/issues/139) .. may be we should give [markdown-it-py from the executablebooks project](https://github.com/executablebooks/markdown-it-py) a try, looks more elaborate to me .. and AFAIK it the same parser that use by the myst-parser (we use the myst-parser in our Sphinx-doc to parse the markdown from info pages)

File: searx/infopage/en/about.md [hunk 4]
```diff
@@ -4,39 +4,39 @@ SearXNG is a fork from the well-known [searx] [metasearch engine], aggregating
...
-SearXNG is a [metasearch engine], inspired by the [seeks project].  It provides
+SearXNG is a [metasearch engine], inspired by the [Seeks project].  It provides
 basic privacy by mixing your queries with searches on other platforms without
 storing search data. Queries are made using a POST request on every browser
-(except Chromium-based browsers*).  Therefore they show up in neither our logs,
-nor your url history. In the case of Chromium-based browser users there is an
-exception: searx uses the search bar to perform GET requests.  SearXNG can be
+(except Chromium-based browsers).  Therefore, they show up in neither our logs,
+nor your URL history. In the case of Chromium-based browser users, there is an
+exception: SearXNG uses the search bar to perform GET requests.  SearXNG can be
```
Comment (return42) [inline searx/infopage/en/about.md]: I think we should drop these two sentences ..

"Queries are made using a POST request on every browser (except Chromium-based browsers). Therefore, they show up in neither our logs, nor your URL history. In the case of Chromium-based browser users, there is an exception: SearXNG uses the search bar to perform GET requests." 

I think the common user do not know the difference of HTTP GET & POST .. and BTW it depends on the SearXNG setup if a browser collects a history:

https://github.com/searxng/searxng/blob/fcb626dedd3965de4fa6c95c47a34fdd14348afd/searx/settings.yml#L78-L80

**Comment by return42** [comment]
@Linerly I pushed two commits on top of your branch .. if this is not OK for you, I can remove them.

@Linerly @dalf @mrpaulblack @tiekoetter 

In this two commits I made a review of the "About SearXNG" and the "Search Syntax" page.

Before we are going to translate these pages its worth to have a review, could you please have a look and give some feedback .. thanks a lot :+1:

File: searx/infopage/en/about.md (hunk 1 above)
Comment (Linerly) [reply searx/infopage/en/about.md]: Yes, that's what LanguageTool suggested me at least

**Comment by Linerly** [comment]
Looks fine, fixed some things

**Comment by return42** [comment]
> Looks fine, fixed some things

:+1: 

> Yes, that's what LanguageTool suggested me at least

Just for my interest, what tool do you use?

**Comment by mrpaulblack** [comment]
Its https://languagetool.org/ I am using it too and it works really well IMO :+1:

**Comment by Linerly** [comment]
> Just for my interest, what tool do you use?

The [LanguageTool online editor](https://languagetool.org/editor/new)

**Comment by return42** [comment]
> The Markdown parser we use ATM does not support Smart punctuation / Simple typographic replacements

FYI: I send PR https://github.com/searxng/searxng/pull/1060 .. with we can use typographic markups (not supported by the current mistletoe parser).

**Comment by dalf** [comment]
#1060 has been merged.
Does this PR need to be updated? Or is it ready to be merged?

<details>
<summary>Here a suggestion for the about page</summary>

Notes:
* Feel free to ignore / copy paste / whatever.
* Woogle has a detail description to set the search engine: https://github.com/benbusby/whoogle-search#set-whoogle-as-your-primary-search-engine

---

# About SearXNG
SearXNG is a fork from the well-known [searx](https://github.com/searx/searx) [metasearch engine](https://en.wikipedia.org/wiki/Metasearch_engine), aggregating the results of other [search engines](https://6bcd-37-166-29-249.eu.ngrok.io/preferences) while not storing information about its users.

## Why use it?
SearXNG may not offer you as personalized results as Google, but it doesn’t generate a profile about you.
SearXNG doesn’t care about what you search for, never shares anything with a third party, and it can’t be used to compromise you.
SearXNG is free software, the code is 100% open, and you can help make it better. See more on [SearXNG sources](https://github.com/searxng/searxng).
If you do care about privacy, want to be a conscious user, or otherwise believe in digital freedom, make SearXNG your default search engine or run it on your own server.

## Technical details - How does it work?
SearXNG is a [metasearch engine](https://en.wikipedia.org/wiki/Metasearch_engine), inspired by the [seeks project](https://beniz.github.io/seeks/). It provides basic privacy by mixing your queries with searches on other platforms without storing search data. 

By default, your queries are sent to SearXNG using POST requests; therefore, they show up in neither our logs nor your URL history. However, there are two exceptions where SearXNG uses the GET method:
* the search bar on Chromium-based browser on the desktop.
* all the requests on a Chromium-based browser on Android.

You can add SearXNG to your browser’s search bar and set it as the default search engine.

## How to set it as the default search engine?
SearXNG supports [OpenSearch](https://github.com/dewitt/opensearch/blob/master/opensearch-1-1-draft-6.md). For more information on changing your default search engine, see your browser’s documentation:
* [Firefox](https://support.mozilla.org/en-US/kb/add-or-remove-search-engine-firefox)
* [Microsoft Edge](https://support.microsoft.com/en-us/help/4028574/microsoft-edge-change-the-default-search-engine)
* Chromium-based browsers [only add websites that the user navigates to without a path.](https://www.chromium.org/tab-to-search)

## Where to find anonymous usage statistics of this instance?
[Stats page](https://6bcd-37-166-29-249.eu.ngrok.io/stats) contains some useful data about the engines used.

## How can I make it my own?
SearXNG appreciates your concern regarding logs, so take the code from the [SearXNG project](https://github.com/searxng/searxng) and run it yourself!
Add your instance to this [list of public instances](https://searx.space/) to help other people reclaim their privacy and make the Internet freer! The more decentralized the Internet is, the more freedom we have!
## Where can I contribute to the translations?
We use [Weblate](https://weblate.bubu1.eu/projects/searxng/searxng/) to translate the user interface.
We use markdown pages to translate the information page (like this one). See the [git repository](https://github.com/searxng/searxng/tree/master/searx/infopage).

## Where are the docs & code of this instance?
See the [SearXNG docs](https://docs.searxng.org/) and [SearXNG sources](https://github.com/searxng/searxng).

</details>

**Comment by return42** [comment]
> https://github.com/searxng/searxng/pull/1060 has been merged.

this PR is independent from 1060

> Does this PR need to be updated? Or is it ready to be merged?

PR is ready to merge.

> Here a suggestion for the about page

If you want something changed, please commit on top of the branch.

> Woogle has a detail description to set the search engine

The links we have are about OpenSearch (the XML file) .. the most descriptions at whoogle are about adding a search for `%s` .. then the user does not have autocomplete .. thats why I think we should not recommend this way.

**Comment by dalf** [comment]
> the most descriptions at whoogle are about adding a search for %s .

I just try on Firefox Android, I confirm https://github.com/benbusby/whoogle-search/pull/119 : OpenSearch doesn't work on FF Android. I haven't tried the other methods.

The section [about FF contains](https://github.com/benbusby/whoogle-search/blob/main/README.md#using-with-firefox-containers) is also interresting.
